<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>secondDialog</class>
 <widget class="QDialog" name="secondDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>571</width>
    <height>653</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="taskText">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>211</width>
     <height>81</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="dueText">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>211</width>
     <height>101</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="labelText">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>390</y>
     <width>211</width>
     <height>101</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="taskButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>570</y>
     <width>181</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Submit</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="priorityText">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>211</width>
     <height>111</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="taskTypeComboBox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>110</y>
     <width>82</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>30</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Task</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>200</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Due Date</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>310</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Priority</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>440</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Label</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>110</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Task Type</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="locationText">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>500</y>
     <width>211</width>
     <height>61</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>510</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Location</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
